packages/expo-maps/android/src/main/java/expo/modules/maps/Records.kt

```diff
@@ -229,11 +236,13 @@ data class MapPropertiesRecord(
   val minZoomPreference: Float = 3.0f
 ) : Record {
   fun toMapProperties(): MapProperties {
+    val mapStyleOptionsParsed = mapStyleOptions?.json?.let { MapStyleOptions(it) }
```

Comment:
the JSON parsing happens inside of compose view, so it doesnt throw an error in case of invalid json.